packages/components/src/components/animations/DeviceAnimation.tsx

```diff
@@ -32,6 +32,7 @@ type DeviceAnimationProps = {
     isOldT2B1Packaging?: boolean;
     deviceUnitColor?: number;
     className?: string;
+    sizeVariant?: 'LARGE';
```

Comment:
I would rename it and wrote the type so that only if you pass `rotate=true` you could also set `rotationSizeVariant` to `LARGE` or leave it undefined.

Reply:
Rather, we put here `imageMode: "STATIC" | "ROTATATIONAL"`